tw-NEDa-backend/src/main/java/org/neda/entity/Population.java

```diff
@@ -50,19 +60,48 @@ public void setAgeDistribution(String ageDistribution) {
         this.ageDistribution = ageDistribution;
     }
 
+    /*
```

Comment:
Remove commented code. Make sure you generate the getters & setters using your IDE and you don't write them yourself.